rewrite-java/src/main/java/org/openrewrite/java/StandardizeNullabilityAnnotations.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.internal.lang.NonNullApi;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+
+@Value
+@NonNullApi
+@EqualsAndHashCode(callSuper = false)
+public class StandardizeNullabilityAnnotations extends Recipe {
+
+    private static final Set<String> KNOWN_NULLABLE_ANNOTATIONS = Stream.of(
+            "javax.annotation.Nullable",
+            "jakarta.annotation.Nullable",
+            "org.eclipse.jdt.annotation.Nullable",
+            "org.jetbrains.annotations.Nullable",
+            "org.netbeans.api.annotations.common.NullAllowed",
+            "androidx.annotation.Nullable",
+            "android.support.annotation.Nullable",
+            "org.checkerframework.checker.nullness.qual.Nullable",
+            "edu.umd.cs.findbugs.annotations.Nullable",
+            "org.springframework.lang.Nullable",
+            "org.jmlspecs.annotation.Nullable",
+            "org.openrewrite.internal.lang.Nullable",
+            "lombok.NonNull"
```

Comment:
This last one looks like it belongs in the other set. Here is a nice list of other similar annotations: https://checkerframework.org/manual/#nullness-related-work. I don't think we need all of these, but the JSpecify annotations would be nice to add, as that framework has gained quite a bit of traction.

Reply:
That's another good source. Thank you for pointing out! I will have a look on them in detail.